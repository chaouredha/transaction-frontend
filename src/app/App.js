import styled from 'styled-components'
import React from 'react'

const AppContainer = styled.div`
  background-color: #eee8e1;
  width: 100vw;
  height: 100vh;
  overflow-y: auto;
  display: flex;
  flex-direction: column;
  font-family: 'Arvo', Arial, sans-serif;
`
const App = () => (
  <AppContainer>
    <h1>Gestion contrat</h1>
  </AppContainer>
)

export default App
