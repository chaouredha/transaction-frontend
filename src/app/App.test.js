import React from 'react'
import { render } from '@testing-library/react'
import 'jest-styled-components'
import { describe, it } from '@jest/globals'
import App from 'app/App'

describe('Notification', () => {
  it('displays Notification', () => {
    const { asFragment } = render(
      <App>
        contenu
      </App>
    )

    expect(asFragment()).toMatchSnapshot()
  })
})
