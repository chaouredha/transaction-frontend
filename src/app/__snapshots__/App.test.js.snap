// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification displays Notification 1`] = `
<DocumentFragment>
  .c0 {
  background-color: #eee8e1;
  width: 100vw;
  height: 100vh;
  overflow-y: auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  font-family: 'Arvo',Arial,sans-serif;
}

<div
    class="c0"
  >
    <h1>
      Gestion contrat
    </h1>
  </div>
</DocumentFragment>
`;
